template/add_funcs.go

```diff
@@ -49,9 +49,19 @@ func (t *Template) addFuncs() {
 	add(Git, t.addGitFuncs)
 }
 
-// Apply all regular expressions replacements to the supplied string
-func (t *Template) substitute(content string) string {
-	return utils.Substitute(content, t.substitutes...)
+// Apply regular expression that match the replacerTimingFilter (default = "")
+//
+// IMPORTANT: The filter is optional, but since there's no such thing in Go, it's a var arg
```

Comment:
I would really prefer to have optional arguments in go. But since it is not available. I don't tend to rely on variable length arguments to implement that. Especially for a private function.

It uselessly adds complexity. In that case, is is preferable to implement the last argument as an enum with possible value (SubstituteBegin and SubstituteEnd).

Reply:
Replaced with the `NONE` ""enum"" value for cases where we don't specify one